mesheryctl/internal/cli/root/connections/view.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2024 Meshery Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/components"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/system"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/server/models/connections"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"io"
+	"net/http"
+)
+
+var viewConnectionCmd = &cobra.Command{
+	Use:   "view",
+	Short: "View a connection id",
+	Long: `View
+a connection detail by id`,
```

Comment:
Can you make this more descriptive, make it more helpful for user by provider what user will expect from this command as output, what are necessary flags to provider, any important thing user should know about this command?

Reply:
done